includes/Modules/Subscribe_With_Google.php

```diff
@@ -47,7 +47,71 @@ public function register() {
 			return;
 		}
 
-		// TODO: Bring back SwG functionality after #3120 is merged.
+		// Register "access" meta field.
+		register_post_meta(
+			'',
+			'sitekit__reader_revenue__access',
+			array(
+				'show_in_rest' => true,
+				'single'       => true,
+				'type'         => 'string',
+			)
+		);
+
+		// Add "Set access to..." bulk edit option.
+		add_filter(
+			'bulk_actions-edit-post',
+			function( $bulk_actions ) {
+				$bulk_actions['sitekit-swg-access'] = __( 'Set access to...', 'google-site-kit' );
```

Comment:
Nit-pick, but I think WordPress generally prefers `&hellip:` instead of `...`.

Reply:
Good to know, updated